Global/Enums.h

```diff
@@ -28,11 +28,6 @@ CLANG_DIAG_OFF(deprecated)
 CLANG_DIAG_ON(deprecated)
 
 NATRON_NAMESPACE_ENTER
-#ifdef SBK_RUN
```

Comment:
why remove?

Reply:
Newer C++ implementations let the preprocessor to expand the name of a namespace if it was defined. Wasn't portable to begin with.